### DISCUSSION TITLE
Archetypes-mapper error due to building names

### DISCUSSION THREAD

**Discussion by metwaly-ali** [architecture-building-systems/CityEnergyAnalyst#3235] [Q&A]
![image](https://user-images.githubusercontent.com/110869501/184463847-9f0318a9-622f-44f4-9abf-bcea6a9f3af4.png)

**Comment by MatNif** [comment]
Hi @metwaly-ali

I saw that you posted a number of issues lately all having more or less the same format. Unfortunately with just a screenshot and a very unspecific question we can't do much to help you. When you post issues in the future could you please follow the following structure:
- Describe the case you are trying to run
- Describe the steps you performed leading up to the error
- Provide the detailed error message that is CEA prompts
- Provide input files that are located in the input folder of your script

If you have watched all the [tutorials](https://www.cityenergyanalyst.com/video-tutorials) on the CEA website and keep having issues with running your case we could also talk about scheduling a zoom call to see why your runs are failing.

**Comment by metwaly-ali** [comment]
ok ........1 - ( Describe the case you are trying to run ) : I want work run script to Archetypes Mapper for the data on the program so I can change the numbers to fit my project.
2- ( Describe the steps you performed leading up to the error ) : Enter Data ( shapefiles ( road - zone - surrounfdin - terrian ) and then Data management and then Archetypes Mapper to run script .
3 - Provide the detailed error message that is CEA prompts  ( jobID: 1 - archetypes-mapper
status: ERROR
parameters:
{
  "scenario": "C:\\Users\\User\\dr.koload\\aswan",
  "input-databases": [
    "comfort",
    "architecture",
    "air-conditioning",
    "internal-loads",
    "supply",
    "schedules"
  ],
  "buildings": [
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential",
    "residential"
  ]
} ) .
 4 - Provide input files that are located in the input folder of your script ( https://drive.google.com/file/d/1L3lhu09DlyF8WYAdWTIg4UK-bM-ViPjv/view?usp=sharing )

**Comment by MatNif** [comment]
Hi @metwaly-ali ,

Thank you for the detailed description. The issue seems to lie with the naming of your buildings.
Currently all of your buildings have the same name (i.e. "residential"). However, in all of its scripts, CEA expects building names to be unique, as it uses them as handle for fetching and manipulating different objects.
The typical naming convention for buildings in CEA is "BXXXX" with X = {0,1,...,9} (e.g. "B1004"). You could try to change your building names to "B1000" - "B1131" (either through the attribute table of whatever GIS-program you are using to manipulate your shapefile or directly in the zone03.dbf file) and try running the archetypes-mapper again.

Hope that helps.

**Reply by metwaly-ali** [reply]
please,when i was working Run Script ( Building Solar radiation ) ............This message appears to me ( Path does not exist ) 
![image](https://user-images.githubusercontent.com/110869501/186375894-73e0a3a3-e691-4946-aac9-1f68cd4ee50e.png)

**Reply by MatNif** [reply]
Hi @metwaly-ali 

Please look for the folder where City Energy Analyst is installed on your computer. Once you find that folder you should find the daysim binaries under Dependencies -> Daysim. Once you find the Daysim folder replace the 'daysim-bin-directory' parameter here with the directory of that folder.

**Reply by metwaly-ali** [reply]
I have already done this, but the problem still exists and this message appears to me...( 
status: ERROR
parameters:
{
  "scenario": "C:\\Users\\User\\Aswan project\\urban form",
  "multiprocessing": true,
  "buildings": [
    "B1000",
    "B1001",
    "B1002",
    "B1003",
    "B1004",
    "B1005",
    "B1006",
    "B1007",
    "B1008",
    "B1009",
    "B1010",
    "B1011",
    "B1012",
    "B1013",
    "B1014",
    "B1015",
    "B1016",
    "B1017",
    "B1018",
    "B1019",
    "B1020",
    "B1021",
    "B1022",
    "B1023",
    "B1024",
    "B1025",
    "B1026",
    "B1027",
    "B1028",
    "B1029",
    "B1030",
    "B1031",
    "B1032",
    "B1033",
    "B1034",
    "B1035",
    "B1036",
    "B1037",
    "B1038",
    "B1039",
    "B1040",
    "B1041",
    "B1042",
    "B1043",
    "B1044",
    "B1045",
    "B1046",
    "B1047",
    "B1048",
    "B1049",
    "B1050",
    "B1051",
    "B1052",
    "B1053",
    "B1054",
    "B1055",
    "B1056",
    "B1057",
    "B1058",
    "B1059",
    "B1060",
    "B1061",
    "B1062",
    "B1063",
    "B1064",
    "B1065",
    "B1066",
    "B1067",
    "B1068",
    "B1069",
    "B1070",
    "B1071",
    "B1072",
    "B1073",
    "B1074",
    "B1075",
    "B1076",
    "B1077",
    "B1078",
    "B1079",
    "B1080",
    "B1081",
    "B1082",
    "B1083",
    "B1084",
    "B1085",
    "B1086",
    "B1087",
    "B1088",
    "B1089",
    "B1090",
    "B1091",
    "B1092",
    "B1093",
    "B1094",
    "B1095",
    "B1096",
    "B1097",
    "B1098",
    "B1099",
    "B1100",
    "B1101",
    "B1102",
    "B1103",
    "B1104",
    "B1105",
    "B1106",
    "B1107",
    "B1108",
    "B1109",
    "B1110",
    "B1111",
    "B1112",
    "B1113",
    "B1114",
    "B1115",
    "B1116",
    "B1117",
    "B1118",
    "B1119",
    "B1120",
    "B1121",
    "B1122",
    "B1123",
    "B1124",
    "B1125",
    "B1126",
    "B1127",
    "B1128",
    "B1129",
    "B1130"
  ],
  "use-latest-daysim-binaries": true,
  "albedo": "0.2",
  "number-of-cpus-to-keep-free": "1",
  "debug": false,
  "daysim-bin-directory": "C:\\Users\\User\\Documents\\METO WORK\\CityEnergyAnalyst\\Dependencies\\Daysim\\bin",
  "n-buildings-in-chunk": "100",
  "write-sensor-data": true,
  "roof-grid": "10",
  "walls-grid": "200",
  "zone-geometry": "2",
  "surrounding-geometry": "5",
  "consider-floors": true,
  "consider-intersections": false,
  "rad-ab": "4",
  "rad-ad": "512",
  "rad-as": "32",
  "rad-ar": "20",
  "rad-aa": "0.15",
  "rad-lr": "8",
  "rad-st": "0.5",
  "rad-sj": "0.7",
  "rad-lw": "0.05",
  "rad-dj": "0.7",
  "rad-ds": "0",
  "rad-dr": "0",
  "rad-dp": "32"
}

**Comment by ShiZhongming** [comment]
Hi @metwaly-ali 
Have it eventually worked? or do you have an update on the error?
